ouroboros-consensus/changelog.d/20230228_130303_jasataco_enforce_bundle_constraints.md

```diff
@@ -0,0 +1,22 @@
+<!--
+A new scriv changelog fragment.
+
+Uncomment the section that is right (remove the HTML comment wrapper).
+-->
+
+<!--
+### Patch
+
+- A bullet item for the Patch category.
+
+-->
+### Non-Breaking
+
+- Version bounds for the bundle have been fixed
```

Comment:
Why aren't the two changelog entries similar?

Reply:
Because `consensus-cardano` depends on `consensus` but not the other way round